### DISCUSSION TITLE
why have v1.1?    Isn't v.Next enough?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/winget-cli#924] [Q&A] [answered]
![Screenshot 2021-05-04 055244](https://user-images.githubusercontent.com/74822870/116947260-19a53600-ac9e-11eb-8d9c-bce6c581dc97.png)
much like Terminal ,  v.Next can be v2. why have a special and separate v1.1 milestone?

**Comment by denelon** [comment, marked as answer]
We're going to use v.Next to "triage" issues as we move them into the next milestone like "v1.1" or "v1.2".

**Reply by ghost** [reply]
@denelon 
considering that v1.0  milestone section will be obsolete/deleted once released, having a  v1.x section makes sense.
